packages/merge-on-green/src/merge-logic.ts

```diff
@@ -31,6 +31,10 @@ interface Reviews {
   state: string;
 }
 
+interface Comments {
```

Comment:
Single comment rather than plural?

```suggestion
interface Comment {
```